sdp.go

```diff
@@ -393,11 +393,11 @@ func addSenderSDP(
 		sendParameters := sender.GetParameters()
 		for _, encoding := range sendParameters.Encodings {
 			if encoding.RTX.SSRC != 0 {
-				media = media.WithValueAttribute("ssrc-group", fmt.Sprintf("FID %d %d", encoding.SSRC, encoding.RTX.SSRC))
+				media = media.WithValueAttribute("ssrc-group", fmt.Sprintf("FID %d %d", encoding.RTX.SSRC, encoding.SSRC))
```

Comment:
I thought it is primary first and then RTX like the code you had before. Do I remember it wrong?